protoc_plugin/test/extension_generator_test.dart

```diff
@@ -47,11 +55,65 @@ void main() {
     );
     fileGenerator.extensionGenerators.single.generate(writer);
 
-    final actual = writer.emitSource(format: false);
-    expectGolden(actual, 'extension.pb.dart');
-    expectGolden(
-      writer.sourceLocationInfo.toString(),
-      'extension.pb.dart.meta',
+    expectGolden(writer.emitSource(format: false), 'extension');
```

Comment:
If we want to keep this similar to the other goldens, this would be `extension.dart`.